src/page_range.rs

```diff
@@ -0,0 +1,42 @@
+use ffi;
+use glib::translate::*;
+
+pub struct PageRange(ffi::GtkPageRange);
+
+impl PageRange {
```

Comment:
This is not used anywhere right now. But would be used in `gtk_print_settings_get_page_ranges()` and the setter.

Please add those at the same time (and same thing with `[repr(C)]` and directly converting slices via casts here)